README.md

```diff
@@ -35,6 +35,8 @@ All tasks are completed on a real machine.
 
 ### Checking files with checkpatch.pl
 
+minor: missed dependencies for checkpatch
+
```

Comment:
minor: checkpatch actually find 1 minor issue after `pip install --user ply`

Reply:
major: actually checkpatch output highlighted one more memory leak issue in `src/list.c`:
```
#define CHECK(STMT) do {if ((ret = STMT) < 0) goto cleanup; } while (0)

// Custom init and exit methods
static int __init custom_init(void)
{
...
        CHECK(identity_create("YOUR_NAME", 1));
        CHECK(identity_create("YOUR_FRIEND_NAME", 2)); // here in case of -ENOMEM the first entry will not be free-ed,
        // `custom_exit()` will not ever be called if `custom_init()` failed.
...
cleanup:
        return ret;
```